src/explore-education-statistics-common/src/components/ContentHtml.tsx

```diff
@@ -87,7 +87,8 @@ export default function ContentHtml({
         return !url?.includes('explore-education-statistics.service.gov.uk') &&
           typeof node.attribs['data-featured-table'] === 'undefined' ? (
           <a href={url} target="_blank" rel="noopener noreferrer">
-            {text} (opens in a new tab)
+            {text}
+            {url.startsWith('mailto:') ? '' : ' (opens in a new tab)'}
```

Comment:
I wonder if this condition may be better as part of the parent condition?

If it's an external link or NOT a `mailto` (first part of the ternary), render the external link markup which shows "(opens in a new tab)" otherwise, just show the link text.

Reply:
Thanks. Only problem with that is I'd have to duplicate the `<a>` tag. If I've interpreted what you're saying correctly?

Reply:
It's already duplicated by the ternary's negative path on line 94. This is what I had in mind:

```
!url?.includes('explore-education-statistics.service.gov.uk' &&
!url.startsWith('mailto:') &&
typeof node.attribs['data-featured-table'] === 'undefined')
    ? <a href={url} target="_blank" rel="noopener noreferrer">{text}</a>
    : <a href={url}>{text}</a>
```